src/core/jpg.js

```diff
@@ -635,7 +655,8 @@ var JpegImage = (function JpegImageClosure() {
   }
 
   JpegImage.prototype = {
-    parse: function parse(data) {
+    parse(data, options = {}) {
+      const dnlScanLines = options.dnlScanLines || null;
```

Comment:
Why not:

```javascript
parse(data, { dnlScanLines = null }) {
```

Reply:
I've made the suggested change; however I also added some validation here, to ensure that the parameter is consistent with the JPEG specification.